cassandane/Cassandane/Cyrus/HTTPPTS.pm

```diff
@@ -93,6 +93,16 @@ sub set_up
                 $res->content_type('application/json');
                 $res->body(encode_json({ otheruser => [ "group:group co",
                                                         "group:group c" ] }));
+            } elsif ($req->query_parameters->{id} eq 'group:group c') {
+                $res = Plack::Response->new(200);
+                $res->content_type('application/json');
+                $res->body(encode_json({ 'group:group co' => [ "cassandane",
+                                                               "otheruser" ] }));
```

Comment:
This looks like a copy/paste error?  "group c" should only contain "cassandane" (the "c", "co" etc in the group names are bits describing whether **c**assandane and/or **o**theruser are members)

Reply:
Yeah that makes sense except; if you look at the line immediately above, "otheruser" is in both groups!  So I just mirrored that.